### PR TITLE
feat: Batch variable rename

### PR DESCRIPTION
**Pull request by duanyytop**
* Batch variable rename
* Format code with Google Java Format Guide

### CODE REVIEW

**Comment by BaojunCZ** [comment]
How about the other code not formatting with Google Java Format Guide?

**Comment by duanyytop** [comment]
> How about the other code not formatting with Google Java Format Guide?

Google Java Format Guide doesn't be imported in project when project created, so now I will format the code which need be changed and the rest code does not change.

Maybe I can create a new pull request to refactor all files.

File: core/src/main/java/org/nervos/ckb/methods/type/CellInput.java [hunk 1]
```diff
@@ -4,44 +4,23 @@
...
- */
+/** Created by duanyytop on 2019-01-08. Copyright © 2018 Nervos Foundation. All rights reserved. */
 public class CellInput {
 
-    /**
-     * previous_output : {"hash":"0x0000000000000000000000000000000000000000000000000000000000000000","index":4294967295}
-     * unlock : {"args":[],"binary":[1,0,0,0,0,0,0,0],"reference":null,"signed_args":[],"version":0}
-     */
+  @JsonProperty("tx_hash")
+  public String txHash;
```
Comment (BaojunCZ) [inline core/src/main/java/org/nervos/ckb/methods/type/CellInput.java]: https://github.com/nervosnetwork/ckb/pull/533/files#diff-a048830b754aed80272a8bba2a30d2ffR67
That doesn't seem to have changed about OutPoint

Comment (duanyytop) [reply core/src/main/java/org/nervos/ckb/methods/type/CellInput.java]: image

**Comment by duanyytop** [comment]
No， just follow ckb pr 533.

James Chen <notifications@github.com>于2019年4月24日 周三21:52写道：

**Pull request closed without merging**